docs(oracle): clarify slash window (#1813)
umee-network/umee: x/oracle/README.md

- The validator fails to vote within the `reward band` around the weighted median for one or more denominations.

> A `SlashWindow` is a window of time during which validators can miss votes. At the end of this period, the amount of misses are tallied and the proper reward or punishment is carried out.

Reviewer: ```suggestion
A `SlashWindow` is a period duration when we count validators miss votes. At the end of every period, the amount of misses are tallied and the proper reward or punishment is carried out.
```

Author: I think `window of time during which` is more clear - switched "this period" -> "every period"

> During every `SlashWindow` (currently set to 7 days), participating validators must maintain a valid vote rate of at least `MinValidPerWindow` (5%), lest they get their stake slashed (currently set to 0.01%). The slashed validator is automatically temporarily "jailed" by the protocol (to protect the funds of delegators), and the operator is expected to fix the discrepancy promptly to resume validator participation.

Reviewer: let's describe what happens if a validator won't fix & unjail.

> Since the `MinValidPerWindow` is currently set to 5% validator must not miss more than 95% of votes in order to be safe from jailing.

Reviewer: ```suggestion
`MinValidPerWindow` is currently set to 5%. This means validator must not miss more than 95% of votes in order to be safe from jailing.
```
